src/test/java/Homework23.java

```diff
@@ -0,0 +1,24 @@
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pagefactory.HomePage;
+import pagefactory.LoginPage;
+
+public class Homework23  extends BaseTest{
+    @Test
+
+    public void renamePlaylist() {
+        String newPlaylistName = "Test Pro Edited PlayList+";
```

Comment:
Hello,
I have checked your homework 23, you're doing great. Keep up the good work!
Best wishes from the HW team (Khanh, khanh@testpro.io)

But I recommend you to use random library in java to generate newPlaylistName randomly. Because when you hardcode your new Playlist Name, if the new name exists before, the script will fail at the step of showing the success verification popup.
//Example: 
![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/144403954/e96ebdb8-eec4-4f42-94bf-fb4df6e7a3f8)